about.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./about.css" />
+  </head>
+  <body>
+    <!-- Navigation Section Starts -->
+    <header>
+      <ul class="top-nav">
+        <li>
+          <img src="./images/facebook.png" alt="">
+        </li>
+        <li>
+          <img src="./images/twitter.png" alt="">
+        </li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+      <nav>
+        <div class="hamburger open" id="hamburger">&#9776;</div>
+        <a href="./home.html">
+          <img
+          class="nav-logo"
+          src="./images/logo.png"
+          alt="CC Global Summit Logo"
+          />
+        </a>
+        <ul class="nav-items">
+          <li class="nav-item"><a href="./about.html">About</a></li>
+          <li class="nav-item"><a href="">Program</a></li>
+          <li class="nav-item"><a href="">Join</a></li>
+          <li class="nav-item"><a href="">Sponsor</a></li>
+          <li class="nav-item"><a href="">News</a></li>
+          <li class="nav-item"><a href="" id="desktop-campaign">CC Campaign</a></li>
```

Comment:
- [ ] At the moment, there is a whitespace at the top of your navbar which is not in the design, please kindly remove this.
- [ ] Also, it will be better to change the `About` nav link to `Home` to make it more intuitive for users to navigate back home.

![image](https://user-images.githubusercontent.com/43995830/227737987-f0e23b96-b6a9-4122-a279-fa46206f3ed7.png)

Reply:
Done